README.md

```diff
@@ -1,284 +1,305 @@
+- Allows the creation of an audit log to log admin activities.
+- Provides the ability to bulk create verified users (follows the same process as point 1)
+- Triggers the sending of emails to users under various conditions (e.g New user, expiring, audit actions).
+- Manages the account application process for PI_AAD users.
+- Provides MI reporting endpoints, which are used to produce the MI report.
+- Flyway for database modifications via SQL ingestion.
+- Secure/Insecure Mode: Use of bearer tokens for authentication with the secure instance (if desired)
+- OpenAPI Spec/Swagger-UI: Documents and allows users or developers to access API resources within the browser.
+- Integration tests using TestContainers for dummy database operations.
 
-    https://docs.gradle.org/current/userguide/checkstyle_plugin.html
+## Azure B2C integration
 
-    Performs code style checks on Java source files using Checkstyle and generates reports from these checks.
-    The checks are included in gradle's *check* task (you can run them by executing `./gradlew check` command).
+PI_AAD users, excluding third parties, will be stored in Azure B2C. Azure B2C provides OIDC which is used in our frontend to authenticate the users.
 
-  * pmd
+Account management interacts with Azure B2C - storing the user when they are created, updating them when their role changes, or deleting them if they have expired.
 
-    https://docs.gradle.org/current/userguide/pmd_plugin.html
+## System Admin Accounts
 
-    Performs static code analysis to finds common programming flaws. Included in gradle `check` task.
+There is a limit to the number of system admin accounts that can be created. This is set using an environment variable: MAX_SYSTEM_ADMIN_ACCOUNTS.
 
+If this is not set, the default maximum number of accounts is 4. The request will be rejected if an attempt is made that causes it to go over this limit.
 
-  * jacoco
+## List Sensitivity checking
 
-    https://docs.gradle.org/current/userguide/jacoco_plugin.html
+### Verified Users
 
-    Provides code coverage metrics for Java code via integration with JaCoCo.
-    You can create the report by running the following command:
+If a publication is PRIVATE or CLASSIFIED, account management is used to validate whether the user has permission to see the publication.
 
-    ```bash
-      ./gradlew jacocoTestReport
-    ```
+For PRIVATE, as long as the user exists in our database, then they are able to see the publication.
 
-    The report will be created in build/reports subdirectory in your project directory.
+For CLASSIFIED, rules are applied based on the List Type and Provenance of the user. For reference, the provenance is which IDAM the user came from (PI_AAD, CFT_IDAM or CRIME_IDAM). The mappings of which provenance can see which list type is defined in the [enum file](./src/main/java/uk/gov/hmcts/reform/pip/account/management/model/ListType.java)
```

Comment:
This file will not exist after merging in the shared data models PR. Not sure if you want to provide a link to that file in pip-data-models model later, or simply mention it is defined in the ListType enum in the project pip-data-models rather than a link to the file. It is up to you :)
This also happen below